text/0058-view-authenticated-sender-memos.md

```diff
@@ -0,0 +1,46 @@
+- Feature Name: `View Authenticate Sender Memo`
+- Start Date: `2022-11-09`
+- MCIP PR: [mobilecoinfoundation/mcips#0058](https://github.com/mobilecoinfoundation/mcips/pull/58)
+
+# Summary
+[summary]: #summary
+
+[#4 Recoverable Transaction History (RTH)](https://github.com/mobilecoinfoundation/mcips/pull/4) introduced three different memo types for [encrypted memos](https://github.com/mobilecoinfoundation/mcips/pull/3). This proposal specifies three more memo types:
+
+- View Authenticated Sender
+- View Authenticated Sender With Payment Request Id Memo
+- View Authenticated Sender With Payment Intent Id Memo
+
+This feature aims to extend the SenderMemo to allow for senders to use their subaddress view private keys to generate Authenticated Sender Memos
+
+# Motivation
+[motivation]: #motivation
+
+With the introduction of the ViewAccountKey and UnsignedTransactions, generating a SenderMemo for RTH became impossible because of the requirement of the Subaddress Spend Private Key during the creation of the TxOuts for the transaction.
```

Comment:
I don't think it's actually impossible now, we have a plan here: https://github.com/mobilecoinfoundation/mobilecoin/pull/2585

The idea is to refactor the `MemoBuilder` object implementation that currently requires the spend key to make it more abstract, and allow that to be a hardware device connection instead.

The only one who is impacted here is Copper's multiparty compute flow.